Anlab.Mvc/Views/TestItems/Create.cshtml

```diff
@@ -0,0 +1,77 @@
+@model Anlab.Core.Domain.TestItem
+
+<!DOCTYPE html>
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Create</title>
+</head>
+<body>
+
+<form asp-action="Create" id="createForm">
+    <div class="form-horizontal">
+        <h2>Create New Test Item</h2>
+        <hr />
+        <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+        <div class="form-group">
+            <label asp-for="Id" class="col-md-2 control-label"></label>
+            <div class="col-md-10">
+                <input asp-for="Id" class="form-control" style="text-transform:uppercase" />
+                <span asp-validation-for="Id" class="text-danger"></span>
+            </div>
+        </div>
+        <div class="form-group">
+            <label asp-for="Analysis" class="col-md-2 control-label"></label>
+            <div class="col-md-10">
+                <input asp-for="Analysis" class="form-control" />
+                <span asp-validation-for="Analysis" class="text-danger"></span>
+            </div>
+        </div>
+        <div class="form-group">
+            <label asp-for="Category" class="col-md-2 control-label"></label>
+            <div class="col-md-10">
+                <select asp-for="Category" class="form-control" form="createForm">
+                    <option value="Soil">Soil</option>
+                    <option value="Plant">Plant</option>
+                    <option value="Water">Water</option>
+                    <option value="Other">Other</option>
+                </select>
```

Comment:
@laholstege Use the values from here:
https://github.com/ucdavis/Anlab/blob/master/Anlab.Core/Domain/TestItem.cs#L32